types/dvgis_dc-sdk/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "@definitelytyped/dtslint/dt.json",
     "rules": {
+        "npm-naming": false,
```

Comment:
Your package is on npm; the lint is trying to tell you that your types package is named incorrectly.